e2e-tests/tfm/cypress/e2e/journeys/feedback/feedback.spec.js

```diff
@@ -1,11 +1,11 @@
 import relative from '../../relativeURL';
 import pages from '../../pages';
-import partials from '../../partials';
 
 context('User submit feedback on TFM', () => {
   it('feedback should contain correct components and text', () => {
-    pages.landingPage.visit();
-    partials.header.betaBannerHref().should('have.attr', 'target', '_blank');
+    // TODO: After SSO user can't visit landingPage.
+    // pages.landingPage.visit();
+    // partials.header.betaBannerHref().should('have.attr', 'target', '_blank');
```

Comment:
Should we remove this comment and have the `betaBannerHref` check after visiting the feedback page?